packages/core/src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -233,9 +234,11 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
     }
 
     private async _executeWithVaulted(payment: OrderPaymentRequestBody, token: string, shouldSetAsDefaultInstrument: boolean | undefined): Promise<InternalCheckoutSelectors> {
+        const paymentMethod = this._store.getState().paymentMethods.getPaymentMethodOrThrow(payment.methodId);
```

Comment:
I think we could just send this value as argument from calling function.

Reply:
sure!, pulled up